e2e/internal/client/outputs.go

```diff
@@ -0,0 +1,104 @@
+package client
+
+import (
+	"encoding/json"
+	"strings"
+)
+
+type AppShowOutput struct {
+	AppName   string                  `json:"appName"`
+	Type      string                  `json:"type"`
+	Project   string                  `json:"project"`
+	Configs   []AppShowConfigurations `json:"configurations"`
+	Routes    []AppShowRoutes         `json:"routes"`
+	Variables []AppShowVariables      `json:"variables"`
+}
+
+type AppShowConfigurations struct {
```

Comment:
Oooh, fun question. It's just because this e2e package is compiled completely separately from the main package - that's why. The dockerfile just copies the e2e package and refers to the archer binary.